crates/miden-objects/src/note/assets.rs

```diff
@@ -107,6 +108,43 @@ impl NoteAssets {
         padded_assets
     }
 
+    /// Returns all Fungible Assets as an Iterator.
```

Comment:
```suggestion
    /// Returns an iterator over all [`FungibleAsset`]s.
```

I'd phrase it like this. Let's do the same on `iter_non_fungible`.